crates/librqbit/webui/src/components/StatusIcon.tsx

```diff
@@ -0,0 +1,23 @@
+import {
+  MdCheck,
+  MdCheckCircle,
+  MdDownload,
+  MdError,
+  MdOutlineMotionPhotosPaused,
+  MdOutlineUpload,
+} from "react-icons/md";
+
+type Props = {
+  finished: boolean;
+  isDownloading: boolean;
+  error: boolean;
+};
+
+export const StatusIcon = ({ finished, isDownloading, error }: Props) => {
+  const isSeeding = finished && isDownloading;
```

Comment:
you can't be both finished and downloading. You can be "finished" and "live" in backend terms